lib/use_packwerk/private/interactive_cli/use_cases/visualize.rb

```diff
@@ -19,20 +19,38 @@ def perform!(prompt)
               teams = TeamSelector.multi_select(prompt)
               VisualizePackwerk.team_graph!(teams)
             else
-              by_name_or_by_owner = prompt.select('Do you select packs by name or by owner?', ['By name', 'By owner'])
-              if by_name_or_by_owner == 'By owner'
-                teams = TeamSelector.multi_select(prompt)
-                selected_packs = ParsePackwerk.all.select do |p|
-                  teams.map(&:name).include?(CodeOwnership.for_package(p)&.name)
-                end
-              else
-                selected_packs = PackSelector.single_or_all_pack_multi_select(prompt)
+              by_name_or_by_owner = prompt.select(
+                'Do you select packs by name or by owner?',
+                ['By name', 'By owner']
+              )
+
+              selected_packs = get_selected_packs(prompt, select_by: by_name_or_by_owner)
+
+              while selected_packs.empty?
```

Comment:
Could we move this into the team selector and pack selector? That way any part of the interactive CLI where a user is asked to select teams and packs has this safeguard, rather than just in the visualize CLI command.

Reply:
In that case I think we'd end up keeping this file the same and just changing the selectors.